17   layout level one

NorPiz opened the pull request:
Ajoute la structure presque complete en placeholder des différents composant de la pièce du premier niveau.
Les objets en bleu sont les objets décoratifs
Les objets en vert sont les objets avec lequel l'un des deux joueurs pourra intéragir.
Création d'un prefab vitre(de l'armoire) pour la synchroniser sur le réseau (l'ajouter par la suite au NetworkManager)
Création d'un prefab objetbrisevitre(l'objet que l'humain) jettera sur l'armoire pour le synchroniser au réseau (L'ajouter au NetworkManager (DefautPrefabsList))

NorPiz committed:
Ajout du Prefab LevelOneLayOut

Ajout de différent cube, level design des différents objets de la pièce. (distinction entre ce qui est Interactable et ce qui est décor avec une couleur verte(interactable) et bleu (décor))

NorPiz committed:
LayOut Level One

Ajout du Prefab LevelOneLayOut
Ajout de différent cube, level design des différents objets de la pièce. (distinction entre ce qui est Interactable et ce qui est décor avec une couleur verte(interactable) et bleu (décor))

NorPiz committed:
Ajout de décor au niveau de la pièce ou se trouve l'human

Ajout de décor au niveau de la pièce ou se trouve l'human. 2* TableFaut  1* Fauteuil
1*Tableau
Ajustement de l'objetBriseVitre

sophie-nayebi commented:
J'aime bien, tout est très clair
![image](https://github.com/user-attachments/assets/46505997-7006-489f-97b9-6206c0c371ee)

sophie-nayebi commented:
Par contre il faudrait mettre tous les objets verts en prefab afin de faciliter la collaboration entre les gens qui vont les implémenter

NorPiz committed:
Correction suite au commentaire de Sophie

Créeation des prefabs(Interactable) et ajouté au DefaultNetworkPrefabs les prefabs qui devront être synchronisés au réseau.
